src/xdocs/cmdline.xml.vm

```diff
@@ -25,8 +25,9 @@
     <section name="Description">
       <p>
         This document describes how to run Checkstyle using the command line
-        tool. The latest version of Checkstyle can be found at <a
-        href="https://checkstyle.org">https://checkstyle.org</a>.
+        tool. The latest version of Checkstyle can be found at
+        <a href="https://github.com/checkstyle/checkstyle/releases">
+          https://github.com/checkstyle/checkstyle/releases</a>.
```

Comment:
Instead of duplicating what another section already does, why don't we just point them to that section?
Current download latest section: https://checkstyle.org/#Download

Reply:
done.
as separate commit for squash.
still fighting with Github support to allow us protect from FORCE PUSH only master branch, not all branches.